### DISCUSSION TITLE
ico必须48*48！

### DISCUSSION THREAD

**Discussion by JINNIAN666** [tw93/Pake#655] [General]
害的我好苦啊，action放图，本地在线都试了，就是生成不成功，后缀ico和png和jpg也是反复试，太难了。
最后才知道必须是ico后缀（windows版本），最重要的是48*48的像素，不然不生效！！！

**Comment by JINNIAN666** [comment]
我用其他图标48*48又不行了，搞不懂

**Comment by JINNIAN666** [comment]
泪目，不是像素的问题，是我原先的ico是乱改后缀而来，不是真正的ico，需要用格式工厂重新转化出真ico，像素大小是不限制的

**Reply by tw93** [reply]
正解，毕竟系统要求
